From 05821269969753e1fea61668e05ecde66ecb71db Mon Sep 17 00:00:00 2001
From: CI Assistant <yossi.ilkanaev@intel.com>
Date: Wed, 5 Dec 2018 08:58:51 +0100
Subject: [PATCH] Merge pull request #300 in SW_UGW/linux from
 bugfix/UGW_SW-32317-8.x to master

* commit '0f8eac77f8f3000cd8d04f24d97befc43f27665e':
  Fix for same macro value being used in SESSION_FLAG2_CONTAINER and SESSION_FLAG2_DS_MPE_QOS resulting in Container learning problem
---
 include/net/ppa/ppa_api.h  | 1 -
 include/uapi/net/ppa_api.h | 2 ++
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/net/ppa/ppa_api.h b/include/net/ppa/ppa_api.h
index 33be91915406..a083db5ac2b6 100644
--- a/include/net/ppa/ppa_api.h
+++ b/include/net/ppa/ppa_api.h
@@ -111,7 +111,6 @@
 /* Flag to extract bits 23-27 of extmark that are set with TC of Ingress Qs */
 #define SESSION_INGRESS_TC				0x0F800000
 
-#define SESSION_FLAG2_DS_MPE_QOS		0x00008000 //Flag for DS_QOS capab
 
 #define ING_TC_SET_POS					19
 #define EGR_TC_SET_POS					20
diff --git a/include/uapi/net/ppa_api.h b/include/uapi/net/ppa_api.h
index aea7f0df4653..9eefe3b601dc 100755
--- a/include/uapi/net/ppa_api.h
+++ b/include/uapi/net/ppa_api.h
@@ -485,6 +485,8 @@
 #define SESSION_FLAG2_CONTAINER 		0x00008000
 #define SESSION_FLAG2_NON_ETHWAN_SESSION 	0x00010000
 
+#define SESSION_FLAG2_DS_MPE_QOS		0x00020000 /* Flag for DS_QOS capability */
+
 /* Other flags */
 #define FLG_PPA_PROCESSED		0x100	/* this used to mark ecah packets which are processed by ppa datapath driver*/
 #define SESSION_FLAG_TC_REMARK		0x40000000 /*Flag to sepcify bit 30 in extmark which specifies packet classified by iptables when set to 1*/
